docs/core-components.md

```diff
@@ -67,21 +74,20 @@ The `_run_main_loop()` function in the `Agent` class performs a series of activi
 
 The `DecisionMaker` component manages global agent state updates proposed by the skills and processes the resulting ledger transactions.
 
-It is responsible for crypto-economic security and goal management and contains the preference and ownership representation of the agent.
+It is responsible for the agent's crypto-economic interactions and goal management, and it contains the preference and ownership representation of the agent.
```

Comment:
Please leave the word security in there for now. It is in the end responsible for crypto economic security, how successful it is in providing that is a different matter, I agree.